pkg/plugins/golang/v3/scaffolds/internal/templates/api/convertion.go

```diff
@@ -0,0 +1,76 @@
+package api
```

Comment:
All go files must to have the license:

See that the check if failing because of that

> ./test/check-license.sh
> Checking for license header...
> license header checking failed:

Could you please copy and paste from other ones to here?

Reply:
Fixed it at [6947076](https://github.com/kubernetes-sigs/kubebuilder/pull/3201/commits/69470768c3f164e0f0ed1f22376f0a7aecbfa460) and [8d30477](https://github.com/kubernetes-sigs/kubebuilder/pull/3201/commits/8d3047790a191a54bb72aaf34be6acb7a756a85c)